app/src/features/sources/SourceCard.tsx

```diff
@@ -119,70 +129,72 @@ const SourceCard = ({ source }: SourceCardProps): JSX.Element => {
             >
               <MoreIcon />
             </IconButton>
-            <Menu
-              anchorEl={anchorEl}
-              open={Boolean(anchorEl)}
-              onClose={handleClose}
-              getContentAnchorEl={null}
-              variant="menu"
-              anchorOrigin={{
-                vertical: "bottom",
-                horizontal: "left",
-              }}
-              transformOrigin={{
-                vertical: "top",
-                horizontal: "left",
-              }}
-            >
-              <MenuItem onClick={handleManagePermissions} disabled>
-                <ListItemIcon className={classes.listItemIcon}>
-                  <ManagePermissionsIcon />
-                </ListItemIcon>
-                <ListItemText primary={t("managePermissions")} />
-              </MenuItem>
-              <MenuItem onClick={handleRenameSource}>
-                <ListItemIcon className={classes.listItemIcon}>
-                  <EditIcon />
-                </ListItemIcon>
-                <ListItemText primary={t("rename")} />
-              </MenuItem>
-              <Divider />
-              <MenuItem onClick={handleDeleteSource}>
-                <ListItemIcon
-                  className={clsx(classes.listItemIcon, classes.delete)}
-                >
-                  <DeleteIcon />
-                </ListItemIcon>
-                <ListItemText
-                  primary={t("delete")}
-                  className={classes.delete}
+          }
+        />
+        <CardContent>
+          {isSourceInfosLoading ? (
+            <CircularProgress />
+          ) : (
+            <>
+              {undefined !== mappingsCount && (
```

Comment:
nit: I prefer 
```suggestion
              {mappingsCount !== undefined && (
```

Reply:
I saw a lot of codebases doing that this way. I'm also curious on knowing if there's a reason behind it.

Reply:
ooh I get it ! It's to avoid accidental assignment.. `if (NULL == foo)` safer than `if (foo == NULL)` which could be `if (FOO = NULL)`

Reply:
I've been told that it was faster this way

Reply:
> I've been told that it was faster this way
I value readability more than speed (in this particular case, I think it's impossible to notice the diff)
 >  It's to avoid accidental assignment
yeah why not, but you must be very tired to make a mistake like this, since in the new versions of JS you have to do use TRIPLE `=` for comparison and the linter warns you if you only use 2.

I don't have a super strong opinion on this so chose what you think is best but I'm personnaly not convinced lol